src/metrics/loc.rs

```diff
@@ -541,6 +543,25 @@ mod tests {
         );
     }
 
+    #[test]
+    fn python_string_on_new_line() {
+        // More lines of the same instruction were counted as blank lines
+        check_metrics!(
+            "capabilities[\"goog:chromeOptions\"][\"androidPackage\"] = \\
+                \"org.chromium.weblayer.shell\"",
+            "foo.py",
+            PythonParser,
+            loc,
+            [
+                (sloc, 2, usize),
+                (ploc, 2, usize),
```

Comment:
Should ploc be 1 too?

Reply:
No, `ploc` is correct because we have two physical lines and one single instruction (so `lloc` is 1 in this case)